core/tests/protractor_desktop/coreEditorAndPlayerFeatures.js

```diff
@@ -243,6 +243,108 @@ describe('Core exploration functionality', function() {
     users.logout();
   });
 
+  it('should be able to edit title', function() {
+    users.createUser('user6@stateEditor.com', 'user6StateEditor');
+    users.login('user6@stateEditor.com');
+
+    workflow.createExploration();
+    explorationEditorPage.navigateToSettingsTab();
+
+    explorationEditorSettingsTab.expectTitleToBe('');
+    explorationEditorSettingsTab.setTitle('title1');
+    explorationEditorPage.navigateToMainTab();
+    explorationEditorPage.navigateToSettingsTab();
+    explorationEditorSettingsTab.expectTitleToBe('title1');
+
+    users.logout();
+  });
+
+  it('should be able to edit goal', function() {
+    users.createUser('user7@stateEditor.com', 'user7StateEditor');
+    users.login('user7@stateEditor.com');
+
+    workflow.createExploration();
+    explorationEditorPage.navigateToSettingsTab();
+
+    explorationEditorSettingsTab.expectObjectiveToBe('');
+    explorationEditorSettingsTab.setObjective('It is just a test.');
```

Comment:
Can we also add a test for "exploration goal with length < 15 shows warning"?
![image](https://user-images.githubusercontent.com/16653571/55673543-f34a1c00-58c6-11e9-89b8-c680895100f6.png)

Reply:
Done. PTAL.
I also found the pre-submission linting check might be improved. I used `"` instead of `'`by accident just now, but it still passed the test. In the coding style document, it states that we prefer `'`. Should I raise an issue to address this?

Reply:
@sophiewu6, Thanks for pointing this, we are working on this issue (#6555).

Reply:
@sophiewu6, FYI the linter issues are fixed now.

Reply:
@DubeySandeep Great! :) Is there any other errors in this PR?